layouts/partials/favicon.html

```diff
@@ -1,5 +1,6 @@
+{{- $s := .Site.Params }}
 <link rel="apple-touch-icon" sizes="180x180" href='{{ absURL "icons/apple-touch-icon.png" }}'>
```

Comment:
I will not accept this commit because it only covers one use case. If you look closer, lines 2, 3 & 5 contain references to different files of the favicon that are used by different platforms to show the website icon & favicon. Add a variable only for line 3 wouldn't be sufficient.

Beyond, that I think it would be better to add a reference on the  readme detailing:
1. What favicon files are required
2. How to generate those files
3. Where to store those files.